content/en/docs/creating-addons/top-level-operator/addon-package.md

```diff
@@ -44,3 +44,19 @@ respectively. *To have these values inject you must maintain the default naming
 All the values in `.spec.config.addonsv1` can be injected into the objects contained in your packageImage. See the
 [package operator documentation](https://package-operator.run/docs/guides/packaging-an-application/#go-templates) to see
 how to do this.
+
+## Tenants Onboarding Steps 
+
+Teams can generate the packageImage themselves using the [package operator documentation](https://package-operator.run/docs/guides/packaging-an-application/#build--validate) although we recommend teams to use Managed Tenants Bundles (MTB) facilities. 
+
+Below are the Steps for generating the packageImage using MTB flow for reference-addon packageImage: 
+
+In MTB, a team just has to create a "package" directory:
+https://gitlab.cee.redhat.com/service/managed-tenants-bundles/-/tree/main/addons/reference-addon/package and add the manifests there, alongside the manifests.yaml[PackageManifest] .
+
+MTB CI will create the packageImage (in addition to the OLM Index Image which is also part of the Team's addon folder https://gitlab.cee.redhat.com/service/managed-tenants-bundles/-/tree/main/addons/reference-addon) .
+
+Then MTB CI raises a MR to [managed-tenants repository](https://gitlab.cee.redhat.com/service/managed-tenants/-/blob/main/addons/reference-addon/addonimagesets/stage/reference-addon.v0.10.1.yaml#L24) , adding a new AddonImageSet with those images.
```

Comment:
We can write something like, `Once the MTB Pr is merged and builds are successful a MR is generated...`